ethereum/yearn/view_yearn_harvests.sql

```diff
@@ -18,31 +18,28 @@ CREATE VIEW yearn."view_yearn_harvests" AS(
     yct."yvault_deposit_token_symbol"
   FROM
     yearn."yvault_strat_evt_Harvested" hvst 
-  LEFT JOIN yearn."view_yearn_contract_strategy" cs on hvst."contract_address" = cs."strategy"
-  LEFT JOIN yearn."view_yearn_contract_tokens" yct on cs."yvault_contract" = yct."yvault_contract"
+  LEFT JOIN yearn."view_yearn_contract_strategy" cs ON hvst."contract_address" = cs."strategy"
+  LEFT JOIN yearn."view_yearn_contract_tokens" yct ON cs."yvault_contract" = yct."yvault_contract"
   WHERE
     "yvault_deposit_token_symbol" is not null
 UNION ALL 
 
-(
-SELECT 
-CASE 
-    WHEN hvst."wantEarned" > 0 THEN (hvst."wantEarned"/(10^yct."yvault_deposit_token_decimals"))
-    ELSE (hvst."wantEarned"/(10^yct."yvault_deposit_token_decimals"))*-1 END as profit_loss,
-NULL as debt_payment,
-NULL as debt_outstanding,
-hvst."evt_tx_hash",
-cs."yvault_contract",
-cs."strategy",
-cs."yearn_type",
-yct."yvault_deposit_token",
-yct."yvault_deposit_token_decimals",
-yct."yvault_deposit_token_symbol"
-FROM 
-yearn."yearn_v1_strat_evt_Harvested" hvst 
-LEFT JOIN yearn."view_yearn_contract_strategy" cs on hvst."contract_address" = cs."strategy"
-LEFT JOIN yearn."view_yearn_contract_tokens" yct on cs."yvault_contract" = yct."yvault_contract" 
-WHERE
-"yvault_deposit_token_symbol" is not null
-)
+  (
+  SELECT 
+    NULL as debt_payment,
```

Comment:
i think you should keep the `profit_loss` here but double check what the `wantEarned` looks like and if you need to times `-1` for negative value